textpipe/pipeline.py

```diff
@@ -36,7 +44,7 @@ def __call__(self, raw):
         raw: incoming, unedited text
         """
         doc = Doc(raw, language=self.language, hint_language=self.hint_language)
-        result_dict = dict(zip(self.pipeline, [doc.__getattribute__(x) for x in self.pipeline]))
+        result_dict = dict([(oper.__class__.__name__, oper(doc)) for oper in self._operations])
```

Comment:
`{operation.__class__.__name__: operation(doc) for operation in self._operations}`